cmd/eval.go

```diff
@@ -410,6 +419,19 @@ func setupEval(args []string, params evalCommandParams) (*evalContext, error) {
 		regoArgs = append(regoArgs, rego.ParsedInput(inputValue))
 	}
 
+	schemaBytes, err := readSchemaBytes(params)
+	if err != nil {
+		return nil, err
+	} else if schemaBytes != nil {
```

Comment:
Code-style-wise, I'd go with
```suggestion
	}
	if schemaBytes != nil {
```

Reply:
Done (commit https://github.com/open-policy-agent/opa/pull/3060/commits/2616431346afb485dd41e56d2b5f68100fb41d87)